src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -40,7 +40,7 @@
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
-final class IriConverter implements IriConverterInterface
+final class IriConverter implements ResourceIriConverterInterface
```

Comment:
`ContextAwareIriConverterInterface` if we follow my next comment.

Reply:
See my previous response.